r3bbase/R3BTprevTnext.cxx

```diff
@@ -0,0 +1,168 @@
+/******************************************************************************
+ *   Copyright (C) 2019 GSI Helmholtzzentrum für Schwerionenforschung GmbH    *
+ *   Copyright (C) 2019-2023 Members of R3B Collaboration                     *
+ *                                                                            *
+ *             This software is distributed under the terms of the            *
+ *                 GNU General Public Licence (GPL) version 3,                *
+ *                    copied verbatim in the file "LICENSE".                  *
+ *                                                                            *
+ * In applying this license GSI does not waive the privileges and immunities  *
+ * granted to it by virtue of its status as an Intergovernmental Organization *
+ * or submit itself to any jurisdiction.                                      *
+ ******************************************************************************/
+
+#include "R3BTprevTnext.h"
+#include "FairLogger.h"
+#include "FairRootManager.h"
+#include "FairRunAna.h"
+#include "FairRuntimeDb.h"
+#include "R3BEventHeader.h"
+#include "R3BMSOffsetPar.h"
+#include "R3BSamplerMappedData.h"
+#include "R3BShared.h"
+#include "TClonesArray.h"
+#include "TMath.h"
+#include "TObjArray.h"
+#include "TRandom.h"
+#include <cstdlib>
+#include <iostream>
+constexpr auto CLOCK_PERIOD = 10;    // ns
+constexpr auto ERROR_NO_MS = -30;    // Assigned value for events where there is no MS
+constexpr auto INVALID_TPTN = -10;   // Assigned value for invalid or nonexistent TPrev or TNext
+constexpr auto ERROR_MULTI_MS = -20; // Assigned value for events where there are multiple MS
+constexpr auto INVALID_EVENT = -40;  // Assigned value for events where the MS was not correctly written
+constexpr auto FIRST_LAST_HIT = 0;   // Assigned value for events where the first/last hit is the MS
```

Comment:
any of these constants which are visible to the user should be placed as public static constexpr members of the class in the header.